app/views/teacher_training_adviser/steps/_identity.html.erb

```diff
@@ -24,7 +24,8 @@
 <%= render Content::InsetTextComponent.new(
   header: "Non-UK citizens:",
   text: "If you qualified as a teacher outside the UK, you need English qualified teacher status (QTS) to get an adviser. <a href=\"/non-uk-teachers/teach-in-england-if-you-trained-overseas\">Find out about teaching in England if you qualified outside the UK</a>.",
-  color: "purple"
+  color: "purple",
+  heading_tag: "h2"
```

Comment:
@Sarah-DfE @emmaachesongray Could you have a look at this please and see whether we need to add a separate h2 above the form? Otherwise the form will sit in the Non-UK h2 section which doesn't feel right